src/app/fdctl/config/default.toml

```
@@ -227,7 +227,8 @@ dynamic_port_range = "8900-9000"
     # none is provided, the default is to ask the first entrypoint which
     # replies to the port check described above what our IP address is.
     # If no entrypoints are specified, then this will be defaulted to
-    # 127.0.0.1. 
+    # 127.0.0.1. If you do set this, make sure this is an externally
```

I'd make this a more prescriptive, and this file is double spaced.

```suggestion
    # 127.0.0.1.  If connecting to a public cluster like mainnet or
    # testnet, the IP address or hostname should be publicly routable so
    # it can be reached by other nodes and you should not use an address
    # on a local subnet like 192.168.0.1. 
```

---

Good point. Done